### PR TITLE
Use IgxSelect instead of IgxDropdown #7072

### LINKED ISSUE #7072: Excel style filtering cannot close drop down
## Description  
Custom filter drop down cannot be closed while clicking on the input

 * igniteui-angular version: 9.0.9
 * browser: all

## Steps to reproduce  

1. Open [ESF sample](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/excel_style_filtering.html).
2. Click on filter in `Order Date` column.
3. In the ESF select `Date Filter -> Before`.
4. Click on `Before` input.
5. Click on `Before` input, again.

## Result  
Drop down remains open.

## Expected result  
Drop down should be closed.

## Attachments  
Attach a sample if available, and screenshots, if applicable.

### PR DESCRIPTION
**Pull request by onlyexeption**
Closes #7072 

### Additional information (check all that apply):
 - [x] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [x] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code ([test guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Test-implementation-guidelines-for-Ignite-UI-for-Angular))
 - [ ] This PR includes API docs for newly added methods/properties ([api docs guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Documentation-Guidelines))
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes ([migrations guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Update-Migrations))
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

File: projects/igniteui-angular/src/lib/grids/filtering/excel-style/excel-style-default-expression.component.html [hunk 1]
```diff
@@ -75,3 +57,9 @@
         {{ grid.resourceStrings.igx_grid_filter_operator_or }}
     </span>
 </igx-buttongroup>
+
+<div #overlayOutlet
+     igxOverlayOutlet
+     class="igx-advanced-filter__outlet"
```
Comment (DiyanDimitrov) [inline projects/igniteui-angular/src/lib/grids/filtering/excel-style/excel-style-default-expression.component.html:63]:
```suggestion
```

File: projects/igniteui-angular/src/lib/grids/filtering/excel-style/excel-style-date-expression.component.html [hunk 2]
```diff
@@ -77,3 +57,9 @@
         {{ grid.resourceStrings.igx_grid_filter_operator_or }}
     </span>
 </igx-buttongroup>
+
+<div #overlayOutlet
+     igxOverlayOutlet
+     class="igx-advanced-filter__outlet"
```
Comment (DiyanDimitrov) [inline projects/igniteui-angular/src/lib/grids/filtering/excel-style/excel-style-date-expression.component.html:63]:
```suggestion
```

**Comment by onlyexeption** [comment]
> I expect there will be additional PRs for versions 9.1.x and 9.2.x, if the issue is reproducible there as well?

There will be for 9.1.x only as we don't have 9.2.x as far as I know.

**Comment by IvayloG** [comment]
> > I expect there will be additional PRs for versions 9.1.x and 9.2.x, if the issue is reproducible there as well?
> 
> There will be for 9.1.x only as we don't have 9.2.x as far as I know.

Yes you are correct, there is currently no 9.2.x branch. I was referring to the latest version branch which is effectively **master**, as **9.2.x** will be branched from master at some point. So PRs in 9.0.x, 9.1.x and master.

**Comment by astaev** [comment]
IgSelect behaves pretty same as the old igxDropdown, cannot be closed when icon or arrow are clicked. It can be closed only when the text is clicked.

**Pull request merged**